workflows/illumina.nf

```diff
@@ -84,7 +85,7 @@ workflow ILLUMINA {
 
     // rename contigs after assembly for cleaner downstream steps
     RENAME_CONTIGS (
-        ch_assemblies, "metaspades"
+        ch_assemblies, "" // empty because metaspades already in the filename
```

Comment:
this feels weird...where did metaspades in sample names come from?

Reply:
When metaspades finished assembly it appends "metaspades" to the fasta filename, so when I renamed the contigs if I included the assembler name "metaspades" it looks funny because it's repeated. Whereas `flye` doesn't put the assembler name in fasta assemblies output, and I like to have this information carried through somehow.